book/training/overview.md

```diff
@@ -0,0 +1,64 @@
+Overview
+========
+
+In plain language, training a deep separation model comprises of the following steps:
+
+1. Grab some data for which you know the mixture and sources.
+2. Pass the mixture through the deep model.
+3. Compare the output of the model (an estimated source) with the ground truth known isolated source.
+4. Adjust the parameters of the deep model so that it gets better at separation.
+5. Go back to 1 and repeat until the deep model is deemed "converged".
+
+To those unfamiliar with deep learning, or with audio, the steps above may 
+raise more questions than they answer:
+
+- How do I get data where I know the mixture and the sources?
+- What does it mean to pass a "mixture" through a deep model? What is the actual data?
+- How do I compare the output of the model to the actual isolated source?
+- How do I make the model better, given that information?
+- How do I know when I'm done?!
+
+If all or some of these questions popped into your head, no worries! 
+That's what this section is for! If you already know the answers, then go ahead
+and skip ahead.
+
+## Data
+
+In the last section about handling data, we already looked at how we can use tools like
+Scaper to construct complex mixtures on the fly with ground truth sources. Almost all
+deep learning based methods in source separation are trained with _synthetic_ mixtures, with
```

Comment:
I think we need to standardize some terms here, esp wrt @justinsalamon 's section. There we're saying that "synthetic" mixtures won't work; here we're saying that's the only thing people use. 

Also, I think the term "synthetic" is a bit too close to "synthesized" even though I know they have quite different meanings in our context. Maybe "artificial"??? But what happened to the "incoherent" vs. "coherent" terminology? I quite liked that.

Reply:
That’ll come soon, how about artificial mixtures?